### DISCUSSION TITLE
webapi conf postgresql

### DISCUSSION THREAD

**Discussion by segment11** [ccfos/nightingale#1301] [Q&A] [answered]
Hi，here is my conf file, start webapi faied, db segment:

webapi init fail: failed to connect to `host=/tmp user=root database=`: dial error (dial unix /tmp/.s.PGSQL.5432: connect: no such file or directory)

[DB]
host="${pgNodeIp}" 
port=5432 
user="${pgUser}"
dbname="postgres"
password="${pgPassword}"
sslmode="disable"
DSN="${pgUser}:${pgPassword}@tcp(${pgNodeIp}:5432)/postgres?sslmode=disable"

**Comment by UlricQin** [comment, marked as answer]
Upgrade nightingale and postgres dsn example：
```
DSN="host=%s port=%s user=%s dbname=%s password=%s sslmode=%s"
```
